command/host_create_test.go

```diff
@@ -68,6 +70,30 @@ func (s *createHostSuite) TestParamDefaults() {
 	s.Equal(apimodels.DefaultContainerWaitTimeoutSecs, s.cmd.CreateHost.ContainerWaitTimeoutSecs)
 }
 
+func (s *createHostSuite) TestParseFromFile() {
+	//file for testing parsing from a json file
+	tmpdir, err := ioutil.TempDir("", "evergreen.command.host_create.test")
+	s.Require().NoError(err)
+
+	path := filepath.Join(tmpdir, "example.json")
+	fileContent := map[string]interface{}{
+		"distro":    "myDistro",
+		"scope":     "task",
+		"subnet_id": "${subnet_id}",
+	}
+
+	s.NoError(utility.WriteJSONFile(path, fileContent))
```

Comment:
I'm sort of curious, can you test and see if this works with a YAML file? (i.e. use the same fileContent but with WriteYAMLFile) 
If it doesn't it won't be too much work to add that consideration, and I do think users are maybe more likely to use yaml since that's how configs are already defined.

Reply:
Changed it to readYAML, so that it can accommodate both.